java/src/test/java/io/cucumber/htmlformatter/HtmlFormatterAcceptanceTest.java

```diff
@@ -0,0 +1,64 @@
+import org.junit.jupiter.api.Test;
+
+import java.lang.InterruptedException;
+import java.lang.ProcessBuilder;
+import java.lang.ProcessBuilder.Redirect;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.InputStreamReader;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import static java.nio.file.Files.newDirectoryStream;
+
+class HtmlFormatterAcceptanceTest {
+
+    @Test
+    void it_works() throws IOException {
+        List<Path> paths = new ArrayList<>();
+
+        newDirectoryStream(
+            Paths.get("..", "javascript", "node_modules", "@cucumber", "compatibility-kit", "features")
+        ).forEach((path) -> {
+            String featureName = path.getFileName().toString();
+            try {
+                newDirectoryStream(
+                    Paths.get(path.toString()),
+                    "*.ndjson"
+                ).forEach((ndjsonFilePath) -> {
+
+                    try {
+                        InputStream inputStream = Files.newInputStream(ndjsonFilePath);
```

Comment:
Consider using `try-with-resources`. E.g: 

This will autoclose the stream after it goes out of scope.

```
try (InputStream inputStream = Files.newInputStream(ndjsonFilePath)) {

}                        
```